docs/source/guide/calibrators.md

```diff
@@ -1,12 +1,13 @@
 ---
 jupytext:
+  formats: ipynb,md:myst
```

Comment:
Does this allow for some fancy `jupytext` command to do the conversions automatically without specifying input/output?

Reply:
Yes! It is from the `jupytext --sync` [option](https://jupytext.readthedocs.io/en/latest/using-cli.html).) It's nice because then I don't have to run the VS Code task every time. Whenever either file is saved, the other is automatically updated.

Reply:
Deleted the line with https://github.com/unitaryfund/mitiq/pull/2568/commits/12f5ae6a139125cddb837d5535132f681e0bc724

Reply:
Sorry didn't mean to ask you to remove it, was just asking what it was for. Fine to leave it now that it's gone.